contrib/systemd/atomic-openshift-node.service

```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=Atomic OpenShift Node
 After=docker.service
-After=atomic-openshift-master.service
```

Comment:
I had them in because I thought it made things clearer but it adds additional maintenance cost in keeping everything updated, I'm fine with removing them.